kalite/coachreports/static/js/coachreports/coach_reports/views.js

```diff
@@ -86,9 +87,48 @@ var CoachSummaryView = BaseView.extend({
     template: require("./hbtemplates/landing.handlebars"),
 
     events: {
-        "click #show_tabular_report": "toggle_tabular_view"
+        "click #show_tabular_report": "toggle_tabular_view",
+        "click #topic-list-submit": "set_data_model"
     },
 
+
+    /*
+    this function populates the topic selection list
+    */
+    appendTopicList: function() {
+        //var parseData = this.data_model.get("available_topics");
```

Comment:
I think "available_topics" is more semantic, so would prefer this be changed back.